package.json

```diff
@@ -17,5 +17,11 @@
     "engines": {
         "node": ">=0.1.90"
     },
-    "main": "./lib/index"
+    "main": "lib",
+    "files": [
+        "examples",
+        "lib",
+        "safe.js",
+        "themes"
+    ]
```

Comment:
ReadMe and LICENSE should be there too. I think it's better if we use `.npmignore` instead of this `files` key.